packages/core/bootstrap/src/lib/cache-warmer/epics.ts

```diff
@@ -174,18 +175,40 @@ export const warmupRequestHandler: Epic<AnyAction, AnyAction, any> = (action$, s
     filter(warmupRequested.match),
     // fetch our required state to make a request to warm up an adapter
     withLatestFrom(state$),
-    map(([action, state]) => ({
-      requestData: state.cacheWarmer.subscriptions[action.payload.key],
-      key: action.payload.key,
-    })),
+    map(([action, state]) => {
+      const requestData = state.cacheWarmer.subscriptions[action.payload.key]
+      return {
+        requestData,
+        key: action.payload.key,
+        subscriptions: state.cacheWarmer.subscriptions 
+      }
+    }),
     filter(({ requestData }) => !!requestData),
     // make the request
-    mergeMap(({ requestData, key }) =>
+    mergeMap(({ requestData, key, subscriptions }) =>
       from(
-        requestData.executeFn({
-          id: requestData.childLastSeenById ? WARMUP_BATCH_REQUEST_ID : WARMUP_REQUEST_ID,
-          data: { ...requestData.origin, maxAge: -1 },
-        }),
+        (async () => {
+          const batches = splitIntoBatches(requestData)
+          const requests = []
+          for (const batch of Object.values(batches)) {
+            const data = {
+              ...requestData.origin,
+              ...batch,
+              resultPath: getBatchRequestResultPath(requestData.batchablePropertyPath, subscriptions, batch),
```

Comment:
The result path should fill itself based on the `endpoint`.

If you take this out does it still work?

Reply:
We'll need to set it to `null` or `undefined` as it's being saved in the parent warmer.  The issue is that the path being saved in the parent warmer might not work with the data being sent in a batched request.

Reply:
Yeah, let's filter it out as start or add to the parent warmer.

There's a bigger gap here in having the system handle `resultPath` better. There are some changes that need to be made to the cache. But it's for another PR.